helix-core/src/test/java/org/apache/helix/integration/spectator/TestRoutingTableProviderPeriodicRefresh.java

```diff
@@ -64,6 +62,8 @@
   private static final int PARTITION_NUMBER = 20;
   private static final int REPLICA_NUMBER = 3;
 
+  private static final long REFRESH_PERIOD_MS = 1000L;
```

Comment:
why 1000ms, could 500ms, or even 100ms work? That will reduce the test time.

Reply:
Works in theory. But I prefer to keep the main test logic the same. I don't have a strong motivation to optimize this test since as Kai mentioned, we shall rewrite it eventually. Just want to make it work first.